lib/src/timetable.dart

```diff
@@ -52,13 +61,27 @@ class Timetable<E extends Event> extends StatelessWidget {
   /// out.
   final OnEventBackgroundTapCallback onEventBackgroundTap;
 
+  /// Builder for Week Indicator area.
+  ///
+  /// If it's not provided, or builder returns `null`,
+  /// default `WeekIndicator` widget will be used
+  final HeaderBuilder weekIndicatorBuilder;
```

Comment:
What do you think about renaming this to `rangeIndicatorBuilder`? That makes it clearer that this widget is also used for `VisibleRange`s other than `week`.

(Also applies to `TimetableHeader`)

Reply:
I can rename for sure, I used `weekIndicatorBuilder` since this builder is used only as a replacement for default `WeekIndicator`, that renders always week of the year https://github.com/TatsuUkraine/timetable/blob/da3e90fe4f73396c470b498a6875f4cbd683c247/lib/src/header/timetable_header.dart#L50

But if you insist, I can rename it, np)

Reply:
@JonasWanke

Reply:
Another name was suggested to me: `leadingHeaderBuilder` doesn't make any assumption about the content you put in it.

Reply:
yep, it's better, will rename

Reply:
fixed